frontend/src/metabase/search/containers/SearchApp.unit.spec.tsx

```diff
@@ -0,0 +1,212 @@
+import querystring from "querystring";
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen, waitFor } from "__support__/ui";
+import SearchApp from "metabase/search/containers/SearchApp";
+import { Route } from "metabase/hoc/Title";
+import {
+  setupDatabasesEndpoints,
+  setupSearchEndpoints,
+} from "__support__/server-mocks";
+import {
+  createMockDatabase,
+  createMockSearchResult,
+} from "metabase-types/api/mocks";
+import { SearchResult } from "metabase-types/api";
+
+import { SearchFilters } from "metabase/search/types";
+
+// Mock PAGE_SIZE so we don't have to generate a ton of elements for the pagination test
+jest.mock("metabase/search/containers/constants", () => ({
+  PAGE_SIZE: 3,
+}));
+
+const TRANSLATED_SIDEBAR_NAMES: Record<string, string> = {
+  "All results": "All results",
```

Comment:
This doesn't seem to be tested as it's not in `TEST_ITEMS` as the result is used in
```ts
TRANSLATED_SIDEBAR_NAMES[model]
```

and `model` is from `TEST_ITEMS`